api/src/test/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceTest.java

```diff
@@ -93,48 +93,34 @@ public void testFeaturedCategory_getFeaturedCategoryForFeaturedWorkspace() {
         .hasValue(FeaturedWorkspaceCategory.TUTORIAL_WORKSPACES);
   }
 
-  void mockFeaturedWorkspaces(String namespace, DbFeaturedCategory dbFeaturedCategory) {
+  void mockFeaturedWorkspaces(
+      String namespace, DbFeaturedCategory dbFeaturedCategory, String firecloudUuid) {
     DbWorkspace mockDbWorkspace =
-        new DbWorkspace().setWorkspaceNamespace(namespace).setWorkspaceId(1);
+        new DbWorkspace()
+            .setWorkspaceNamespace(namespace)
+            .setWorkspaceId(1)
+            .setFirecloudUuid(firecloudUuid);
 
     DbFeaturedWorkspace dbFeaturedWorkspace =
         new DbFeaturedWorkspace().setWorkspace(mockDbWorkspace).setCategory(dbFeaturedCategory);
 
-    RawlsWorkspaceResponse rawlsWorkspaceResponse = new RawlsWorkspaceResponse();
-    rawlsWorkspaceResponse.workspace(new RawlsWorkspaceDetails().workspaceId("1"));
-    rawlsWorkspaceResponse.accessLevel(RawlsWorkspaceAccessLevel.OWNER);
-    when(mockFireCloudService.getWorkspace(
```

Comment:
we no longer call getWorkspace() with params - instead we call getWorkspaces() without params.  See new L115.